src/firmware/boards/robot_stm32h7/freertos.c

```diff
@@ -320,29 +377,94 @@ void initIoNetworking(void)
         &robot_log_msg, TbotsProto_RobotLog_fields, MAXIMUM_TRANSFER_UNIT_BYTES);
 
     // TODO (#2064) mainboard rev 2.0 doens't have a reset pin for the u-blox chip
-    // so we can't enable the communicator. Uncomment this for mainboard rev 2.1
-    // which should fix this issue.
-    //
-    // GpioPin_t *ublox_reset_pin =
-    //     io_gpio_pin_create(ublox_reset_GPIO_Port, ublox_reset_Pin, ACTIVE_LOW);
-    // io_ublox_odinw262_communicator_init(&huart8, ublox_reset_pin, 5);
+    // so we can't enable the communicator.
+    GpioPin_t *ublox_reset_pin =
+        io_gpio_pin_create(ID_SEL_4_GPIO_Port, ID_SEL_4_Pin, ACTIVE_LOW);
+
+    // Initialize the ublox communicator
+    io_ublox_odinw262_communicator_init(&huart4, ublox_reset_pin, 2);
 
-    // initialize network logger
+    // Initialize network logger
     io_network_logger_init(RobotLogProtoQHandle);
 }
 
 void initIoDrivetrain(void)
 {
-    // TODO (#2063) the motor driver pins are not wired correctly on mainboard rev2.0
-    // We will not be trying to drive any motors until we have mainboard rev2.1 which
-    // addresses the board issues.
+    // MOTOR WHEEL_BACK_LEFT
+    GpioPin_t *motor_a_reset_pin = io_gpio_pin_create(
+        WHEEL_BACK_LEFT_RESET_GPIO_Port, WHEEL_BACK_LEFT_RESET_Pin, ACTIVE_HIGH);
+    GpioPin_t *motor_a_mode_pin = io_gpio_pin_create(
+        WHEEL_BACK_LEFT_MODE_GPIO_Port, WHEEL_BACK_LEFT_MODE_Pin, ACTIVE_HIGH);
+    GpioPin_t *motor_a_dir_pin = io_gpio_pin_create(WHEEL_BACK_LEFT_DIR_GPIO_Port,
+                                                    WHEEL_BACK_LEFT_DIR_Pin, ACTIVE_HIGH);
+
+    // MOTOR WHEEL_FRONT
+    GpioPin_t *motor_b_reset_pin = io_gpio_pin_create(
+        WHEEL_FRONT_LEFT_RESET_GPIO_Port, WHEEL_FRONT_LEFT_RESET_Pin, ACTIVE_HIGH);
+    GpioPin_t *motor_b_mode_pin = io_gpio_pin_create(
+        WHEEL_FRONT_LEFT_MODE_GPIO_Port, WHEEL_FRONT_LEFT_MODE_Pin, ACTIVE_HIGH);
+    GpioPin_t *motor_b_dir_pin = io_gpio_pin_create(
+        WHEEL_FRONT_LEFT_DIR_GPIO_Port, WHEEL_FRONT_LEFT_DIR_Pin, ACTIVE_HIGH);
+
+    // MOTOR C
```

Comment:
update to dribble and similarly for below?

Reply:
fixed